AppiumSwiftClient/command/W3C/element/File.swift

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
for file commands? (as next change??)

Reply:
Uh, that's clearly a mistake :grimacing:. I remember creating a file and hitting enter before naming it, hence a File.swift file. I will delete it and push it again.